auth/credentials/impersonate/impersonate_test.go

```diff
@@ -25,38 +25,54 @@ import (
 	"time"
 
 	"github.com/google/go-cmp/cmp"
+	"google.golang.org/api/option"
```

Comment:
This module intentionally does not rely on `google.golang.org/api`. This does not look needed for the tests.

Reply:
Fixed, thanks!